photon-client/src/components/dashboard/tabs/ArucoTab.vue

```diff
@@ -2,13 +2,31 @@
 import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
 import { PipelineType } from "@/types/PipelineTypes";
 import CvSlider from "@/components/common/cv-slider.vue";
+import CvRangeSlider from "@/components/common/cv-range-slider.vue";
+import CvSwitch from "@/components/common/cv-switch.vue";
 import { computed, getCurrentInstance } from "vue";
 import { useStateStore } from "@/stores/StateStore";
 
 // TODO fix pipeline typing in order to fix this, the store settings call should be able to infer that only valid pipeline type settings are exposed based on pre-checks for the entire config section
 // Defer reference to store access method
 const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings;
 
+// TODO fix cv-range-slider so that store access doesn't need to be deferred
+const threshWinSizes = computed<[number, number]>({
+  get: () => {
+    if (currentPipelineSettings.pipelineType === PipelineType.Aruco) {
```

Comment:
Is there a reason that `pipelineType` is being checked here? This entire tab cannot be viewed unless the `pipelineType` is Aruco

Reply:
I'm fumbling around with the UI stuff, but I'm pretty sure here that this check is necessary to access the `Aruco` pipeline settings (as seen in the template `v-if="currentPipelineSettings.pipelineType === PipelineType.Aruco"`). Without it, I get
```
Property 'threshWinSizes' does not exist on type 'ActivePipelineSettings'.
  Property 'threshWinSizes' does not exist on type 'ReflectivePipelineSettings'.
```

Reply:
I am able to pass `currentPipelineSettings.threshWinSizes` to `v-model` with no errors. Try and refresh typescript server?